src/views/virtualmachines/utils/virtualMachineRowFilter.ts

```diff
@@ -192,6 +193,28 @@ const useInstanceTypesFilter = (vms: V1VirtualMachine[]): RowFilter => {
   };
 };
 
+const useIPSearchFilter = (vmiMapper: VmiMapper): RowFilter => ({
+  filter: (input, obj) => {
+    const search = input.selected?.[0];
+
+    if (!search) return true;
+
+    const vmi = vmiMapper.mapper?.[obj?.metadata?.namespace]?.[obj?.metadata?.name];
+
+    const ipAddresses = getVMIIPAddresses(vmi);
+
+    if (search.includes('/')) {
+      return ipAddresses.some((ipAddress) => compareCIDR(search, ipAddress));
+    }
+
+    return ipAddresses.some((ipAddress) => ipAddress?.startsWith(search));
+  },
+  filterGroupName: t('IP Address'),
+  isMatch: () => true,
```

Comment:
?

Reply:
typescript gives some errors. `RowFilter`  wants `isMatch` or `reducer` function. I created a PR against SDK to add a new Type `RowSearchFilter`. With that PR merged I can remove this function. Creating in our plugin a new type would break `useListPageFilter` hook anyways so for know there is no solution